src/app/SearchDetail.jsx

```diff
@@ -55,7 +55,7 @@ export default class SearchDetail extends React.Component {
       })
       .then((res) => {
         this.setState({
```

Comment:
Dealing with numerical array indexes like this is not ideal.  I realize that this is a data side thing, however.  I'll revisit how we can better provide search results and we can possibly refactor in the future.

My ultimate goal is to be able to have the search object with results, counts, etc... capable of dropping into the state without any need to 'unpack' it, then make it so we can just send the state back to the server without needing to 'repack' it.